dataset/companies/Web3.yaml

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Let's use a lowercase file name, so it will be `web3.yaml`

Reply:
yeah we can but I followed the way it was used. If we are going to use lowercase, it should be in all files I think 
![image](https://github.com/TechForPalestine/boycott-israeli-tech-companies-dataset/assets/10674070/3d19d2bb-785e-4ffd-8520-34ddcae99fee)

Reply:
well I think the old files don't follow the standard naming convention 
https://www.w3schools.io/file/yaml-naming-conventions/
https://stackoverflow.com/questions/22771226/what-is-the-canonical-yaml-naming-style

This should be fixed but I think not in this PR

Reply:
my bad, let's keep it the way it is